### DISCUSSION TITLE
how to use special character * in intellij editor

### DISCUSSION THREAD

**Discussion by gopi-chowdhary** [learndevopsonline/learndevopsonline#1501] [Q&A]
while using in print used memory---* character getting in down and it is showing has error 
![Uploading Screenshot (6).png…]()

**Comment by sreedharm07** [comment]
Hi @gopi-chowdhary your screen shot is not visible please upload again

**Comment by gopi-chowdhary** [comment]
![Screenshot (6)](https://github.com/learndevopsonline/learndevopsonline/assets/145432552/d385fa75-07fc-4c99-ae10-a1f6b7f75768)

**Comment by gopi-chowdhary** [comment]
workstation IP : 54.90.110.162

![Screenshot (7)](https://github.com/learndevopsonline/learndevopsonline/assets/145432552/e317e2c7-b185-4a9c-b90e-2b0fc8fb1927)

**Reply by sreedharm07** [reply]
Hi @gopi-chowdhary 
error tells that FREE variable is not defined.  error is not because of *
you should save fact(variable) or set fact  in a variable name **FREE** but you saved memory free status into **msg**. thats why its showing variable is undefined error
please modify and try once

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/032ff4d8-8bed-4949-a2b5-40c608820433)

**Reply by gopi-chowdhary** [reply]
tq bro its working
